src/manager.rs

```diff
@@ -8,10 +8,12 @@ use value_classes::value_id::ValueID;
 
 pub struct Manager {
     pub ptr: *mut extern_manager::Manager,
-    options: Options,
     watchers: Vec<Option<Box<WatcherWrapper>>>
 }
 
+unsafe impl Send for Manager {}
+unsafe impl Sync for Manager {}
```

Comment:
I think it's safe because nearly all manager functions are threadsafe and we encapsulate properly (using &mut) the ones that aren't.